plenum/test/replica/stashing/test_stash_future_view.py

```diff
@@ -58,20 +54,12 @@ def test_process_three_phase_msg_and_stashed_future_view(txnPoolNodeSet, looper,
             # (len(txnPoolNodeSet) - 2) - prepare msgs
             # (len(txnPoolNodeSet) - 1) - commit msgs
             stashed_master_messages = 2 * (1 + (len(txnPoolNodeSet) - 2) + (len(txnPoolNodeSet) - 1))
```

Comment:
Why did we remove the assertions? Why don;t they pass anymore?

Reply:
Because backups don't have a view change and don't stash messages in this case.